tests/lib/fde-setup-hook/fde-setup.go

```diff
@@ -157,6 +196,9 @@ func runFdeRevealKey() error {
 func main() {
 	var err error
 
+	// XXX: workaround systemd bug
```

Comment:
What bug is it, can you link to a bug# or elaborate?

Reply:
I can probably remove it, let me try - the bug is reported as https://bugs.launchpad.net/ubuntu/+source/systemd/+bug/1921145 but it's only an issue when the hook fails.

Reply:
well, if people use this as an example they, will hit this, so better to leave it

Reply:
Would be good to enhance the comment to say what bug we're working around here (if we leave XXX).